daemon/Cargo.toml

```
@@ -38,7 +38,7 @@ serde_json = "1"
 serde_with = { version = "1", features = ["macros"] }
 sha2 = "0.10"
 snow = "0.9"
-sqlx = { version = "0.5", features = ["offline", "sqlite", "uuid", "runtime-tokio-rustls"] }
+sqlx = { version = "0.5.13", features = ["offline", "sqlite", "uuid", "runtime-tokio-rustls"] }
```

```suggestion
sqlx = { version = "=0.5.13", features = ["offline", "sqlite", "uuid", "runtime-tokio-rustls"] }
```

To pin it, you will need to use `=`. Without a modifier, `^` is assumed which means update to latest supported version without breaking changes and for < 0.x that means every patch version.

---

See https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility.

---

> To pin it, you will need to use `=`. Without a modifier, `^` is assumed which means update to latest supported version without breaking changes and for < 0.x that means every patch version.

fair enough! in this case, I think I worded the commit poorly. What I meant, is that this is the minimum version, so that dependabot could submit a PR when new one comes out.

---

Yes that is what it will do! But it will already do that because the lock file specifies that 0.5.13 is the current one.

---

the thing is it doesn't already do that for some reason - the lock file was stuck on `0.5.11`, despite sqlx having 2 releases in between. I thought this commit would make dependabot raise PRs when the dependency upgrades are available - am I missing something?

---

I've changed the terminology from "pin" to "specify" to better highlight what I'm doing here. Does that work for you @thomaseizinger ?